packages/@adobe/spectrum-css-temp/components/inputgroup/skin.css

```diff
@@ -120,6 +139,17 @@ governing permissions and limitations under the License.
         }
       }
     }
+
+    &:focus-ring {
+      .spectrum-InputGroup-input {
+        box-shadow: 0 1px 0 var(--spectrum-textfield-quiet-border-color-key-focus);
+      }
+
+      .spectrum-FieldButton {
+        box-shadow: 0 1px 0 var(--spectrum-textfield-quiet-border-color-key-focus);
+        border-color: var(--spectrum-textfield-quiet-border-color-key-focus);
```

Comment:
As per design feedback, only show the box-shadow if the combobox is keyboard focused.